dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/AttachmentsAdapter.java

```diff
@@ -22,12 +22,18 @@
 /**
  * This class provides map adapters to support attachments in RpcContext, Invocation and Result switch from
  * <String, String> to <String, Object>
+ *
+ * please use {@link org.apache.dubbo.common.utils.MapUtils}
+ *
  */
+@Deprecated
 public class AttachmentsAdapter {
 
+    @Deprecated
     public static class ObjectToStringMap extends HashMap<String, String> {
         private Map<String, Object> attachments;
 
+        @Deprecated
```

Comment:
this file method `convert` has problem, if object type is not string, it will return null, cound you fix it conveniently

Reply:
Hmm, the new method takes this into consideration, calling Object#toString() directly。